helpers/tf_cfg.py

```diff
@@ -11,13 +11,33 @@
 __copyright__ = "Copyright (C) 2017-2019 Tempesta Technologies, Inc."
 __license__ = "GPL2"
 
+import logging
+
+from rich import pretty
+from rich.logging import RichHandler
+
+logger = logging.getLogger(__name__)
+
+log_levels = {
+    0: logging.CRITICAL,
+    1: logging.ERROR,
+    2: logging.WARNING,
+    3: 25,
```

Comment:
What does this magic number mean?

Reply:
At this moment, our framework has 7 levels of debug (0-6). So I proposed this scheme. Probably, we should not use the standard library output and will add new level names. For example:
```python3
log_levels = {0: 7, 1: 6, 2: 5, 3: 4, 4: 3, 5: 2, 6: 1}

for k, v in log_levels.items():
    logging.addLevelName(k, f"Lvl {v}")
```

Reply:
What is the reason to have 7 levels? 

I think having such structure `{0: 7, 1: 6, 2: 5, 3: 4, 4: 3, 5: 2, 6: 1}` for levels is not good idea, it may confuse.

Reply:
I don't know. I think this is a very old idea, and no one knows. We must change it when we switch to logging. Probably you should add a comment to this code.